src/app/app.openWeatherMapsService.js

```diff
@@ -0,0 +1,19 @@
+export default class OpenWeatherMapService {
+    constructor($http) {
+        this.$http = $http;
+    }
+    getCurrentWeather(city) {
+        const apiKey = '608c0e7fd9bce774dd414deb10de6e52';
```

Comment:
In this case is not necessary, but for the future is better to have apiKey constant available for the whole service (just moving this line to the top of the file)